src/main/scala/rps/Game.scala

```diff
@@ -0,0 +1,63 @@
+package rps
+
+import scala.io.StdIn.readLine
+import scala.util.Random
+
+object Game {
+
+  // TODO this should be an enum (or Scala equivalent of TS Enum)
+  object Weapon {
+    val Rock: String = "0" // 🪨
+    val Paper: String = "1" // 📄
+    val Scissor: String = "2" // ✂️
```

Comment:
```suggestion
    val Scissors: String = "2" // ✂️
```
(scusa la pignoleria, ma scissors è sempre plurale)

Reply:
LOL grazie